### PR TITLE
Add prameshj and swetharepakula into ingress-gce maintainer

### PR DESCRIPTION
**Pull request by freehan**
/assign bowei@

/cc prameshj swetharepakula

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1523,6 +1523,8 @@ teams:
     - MrHohn
     - rramkumar1
     - thockin
+    - prameshj
+    - swetharepakula
```
Comment (palnabarun) [inline config/kubernetes/org.yaml:1527]: Can you please ensure that the members list is sorted after the addition?

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: config/kubernetes/org.yaml (hunk 1 above)
Comment (freehan) [reply config/kubernetes/org.yaml:1527]: ack

**Pull request merged**